test/autorest/generated/custombaseurlgroup/paths.go

```diff
@@ -44,8 +44,8 @@ func (client *pathsOperations) GetEmpty(ctx context.Context, accountName string)
 // getEmptyCreateRequest creates the GetEmpty request.
 func (client *pathsOperations) getEmptyCreateRequest(accountName string) (*azcore.Request, error) {
 	host := "http://{accountName}{host}"
+	host = strings.ReplaceAll(host, "{host}", client.host)
```

Comment:
I think this is being generated incorrectly...shouldn't it be `client.Client.host`?
Also surprised that the tests pass with this...or am I missing something?

Reply:
The `*Client` is anonymously embedded in `pathsOperations`, so we can access it directly.

Reply:
Ah true, a mistake I made in the previous version. Thanks for fixing it!